core/templates/components/state-editor/state-solution-editor/state-solution-editor.component.ts

```diff
@@ -81,13 +80,11 @@ export class StateSolutionEditorComponent implements OnInit {
     private stateHintsService: StateHintsService,
     private stateInteractionIdService: StateInteractionIdService,
     private stateSolutionService: StateSolutionService,
-    private windowDimensionsService: WindowDimensionsService,
   ) {}
 
   ngOnInit(): void {
     this.correctAnswer = null;
-    this.solutionCardIsShown = (
-      !this.windowDimensionsService.isWindowNarrow());
+    this.solutionCardIsShown = true;
```

Comment:
Please add a check for this in the frontend test.

Reply:
The test for this already exists at `state-solution-editor.component.spec.ts -> Line 117` ([link](https://github.com/Lawful2002/oppia/blob/0d3d9c6bbc53082a083e212648446d6b6e7bef66/core/templates/components/state-editor/state-solution-editor/state-solution-editor.component.spec.ts#L117))